requests_kerberos/kerberos_.py

```diff
@@ -214,7 +213,7 @@ def generate_request_header(self, response, host, is_preemptive=False):
                 username=self.principal,
                 hostname=kerb_host,
                 service=self.service,
-                channel_bindings=self.cbt_struct,
+                channel_bindings=self._cbts.get(host),
```

Comment:
While it does this by default it would be good to specify the default to show we know that this could be `None` when it's not set

```suggestion
                channel_bindings=self._cbts.get(host, None),
```

It can not be populated in the case of `force_preemptive` as that skips the `handle_response` branch.